packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../utils/index';
+
+@mixin iui-expandable-block {
+  --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon);
+  @include iui-expandable-block-variant;
+  @include iui-expandable-block-size;
+
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-radius: var(--iui-border-radius-1);
+
+  &:where(.iui-expanded) {
+    --_iui-expandable-block-expander-icon-transform: rotate(90deg);
+  }
+}
+
+/// @arg $variant Can be one of: default, borderless
+@mixin iui-expandable-block-variant($variant: 'default') {
+  @if $variant == 'default' {
+    --_iui-expandable-block-border: 1px solid var(--iui-color-border);
+    --_iui-expandable-block-background-color: var(--iui-color-background);
+    --_iui-expandable-block-content-padding: var(--iui-size-s);
+    --_iui-expandable-block-header-background-color-disabled: var(--iui-color-background-disabled);
+    --_iui-expandable-block-header-background-color-hover: var(--iui-color-background-hover);
+    --_iui-expandable-block-header-focus-offset: -2px;
+    --_iui-expandable-block-header-focus-thickness: 2px;
+
+    &:where(.iui-expanded) {
+      --_iui-expandable-block-header-bottom-border-radii: 0;
```

Comment:
Why `radii`, not `radius`?

Reply:
That variable is setting both bottom left & bottom right border radius, so I thought it would be appropriate.  I'm fine with changing it though, just let me know if you'd like it changed.

Reply:
It's just a bit strange. CSS has 'radius' for all (eg. border-radius: 10% 30% 50% 70%). 
If it's not already in variables package, I think "radius" would fit better.

Reply:
Changed to `radius`.